internal/logic/logic.go

```diff
@@ -247,9 +244,20 @@ func (l userMgmtSvcLogic) UserData(id any) *respModel.Response {
 			Data:    nil,
 		}
 	}
+
+	eSlice := strings.Split(user[len(user)-1].Email, "@")
+	a := strings.Split(eSlice[0], "")
+	for i := 2; i < len(a)-2; i++ {
```

Comment:
Is there a test cases where the email is like
`abc@gmail.en.com`?

The expected masked email should be `abc@xxx.en.com`

Reply:
done